app/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.2"
```

Comment:
According to **Lewis**:
> **Message**: Old buildToolsVersion 23.0.2; recommended version is 23.0.3 or later.
 **Explanation**: This detector looks for usages of libraries where the version you are using is not the current stable release. Using older versions is fine, and there are cases where you deliberately want to stick with an older version. However, you may simply not be aware that a more recent version is available, and that is what this lint check helps find.